src/components/Header.module.scss

```diff
@@ -0,0 +1,33 @@
+@import "./utils/index.scss";
+.head_block {
+  @include theme($theme: $darkBlue);
+  transition:
+    background-color 0.8s,
+    color 0.8s;
```

Comment:
When you use the same timing for different properties you don't need to write it each time.
```
transition: background-color, color 0.8s;
```

Reply:
ok